t/ui/02-list-group.t

```diff
@@ -17,7 +17,21 @@ use OpenQA::SeleniumTest;
 my $t = Test::Mojo->new('OpenQA::WebAPI');
 driver_missing unless my $driver = call_driver;
 
+my $template = {
```

Comment:
All the changes in this test seem to be leftovers from your previous attempt, right?
Didn't we say that `group_glob=*` should do it? If there's not yet a test for that, you could add one, but I suggest in a different PR.